homeassistant/components/camera/__init__.py

```diff
@@ -221,7 +221,11 @@ async def async_get_mjpeg_stream(
     """Fetch an mjpeg stream from a camera entity."""
     camera = _get_camera_from_entity_id(hass, entity_id)
 
-    return await camera.handle_async_mjpeg_stream(request)
+    try:
+        stream = await camera.handle_async_mjpeg_stream(request)
+    except ConnectionResetError:
+        _LOGGER.debug("Error while writing MJPEG stream to transport")
+    return stream
```

Comment:
There will be a NameError here now if the except case happens.

Reply:
Right, thanks.